### ISSUE TITLE
ReadIS: error in ImportSdkFiles  printf.c path

### ISSUE THREAD

**Issue by Cils** [inertialsense/inertial-sense-sdk#491]
The ImportSdkFiles contains a wrong path for the files printf.c, and printf.h.
There is no ../../../hw-libs/printf-master" folder but a " ../../../hw-libs/printf" folder.
The path in ISConstants.h at line 147 is pointing to "/hw-libs/printf" so I assume that this is the correct path.

**Comment by waltjohnson** [comment]
You are correct.  Thanks for the feedback.  You can pull the latest to find the fix.
